.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "cesium",
+    "rules": {
+        "no-unused-vars": ["off"]
```

Comment:
Shouldn't this be `"no-unused-vars": ["error", {"vars": "all", "args": "none"}],` to maintain what we had?